<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelObject</class>
 <widget class="QWidget" name="LabelObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QLineEdit" name="textLabelName"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="TextFarLabel">
     <property name="text">
      <string>Texte à afficher</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="textDisplayed"/>
   </item>
   <item row="5" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Taille</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Emplacement</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="xposInput">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="yposInput">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="zposInput">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Épaisseur du texte</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="thickInput">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Orientation</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="xorientInput">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="yorientInput">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="sizeInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.750000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textCloseLabel">
     <property name="text">
      <string>Nom de l'étiquette</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textLabelName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thickInput</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xposInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yposInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zposInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xorientInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yorientInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textDisplayed</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LabelObject</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
